tensorly/decomposition/_parafac2.py

```diff
@@ -277,6 +342,9 @@ def _parafac2_reconstruction_error(
     projected_tensor : ndarray, optional
         The projections of X into an aligned tensor for CP decomposition. This can be optionally
         provided to avoid recalculating it.
+    mask : ndarray, optional
+        array of booleans with the same shape as tensor should be 0 where the values are
+        missing and 1 everywhere else.
```

Comment:
```suggestion
        An array with the same shape as the tensor. It should be 0 where there are
        missing values and 1 everywhere else.
```
This text is a bit unclear. Maybe something like this instead? :)

Reply:
Thank you for looking into this. This was addressed in fe0acb5fb91b54d00926258756e732187223b33b.